client/homebrew/pages/accountPage/accountPage.jsx

```diff
@@ -56,7 +55,11 @@ const AccountPage = createClass({
 			<div className='dataGroup'>
 				<h3>Google Information <i className='fab fa-google-drive'></i></h3>
 				<p><strong>Linked to Google: </strong> {this.props.uiItems.googleId ? 'YES' : 'NO'}</p>
-				{this.props.uiItems.googleId ? <p><strong>Brews on Google Drive: </strong> {this.props.uiItems.fileCount || '-'}</p> : '' }
+				{this.props.uiItems.googleId &&
+					<p>
+						<strong>Brews on Google Drive: </strong> {this.props.uiItems.fileCount != '-' ? this.props.uiItems.fileCount : <>Unable to retrieve files - <a href='https://github.com/naturalcrit/homebrewery/discussions/1580'>follow these steps to renew your Google credentials.</a></>}
```

Comment:
Minor change: Instead of passing in `-` from `app.js` when there are 0 google brews, can we pass in 0? Since the `-` doesn't seem like it will be used here anymore, and it might be a source of confusion why we are using a hyphen in this one spot and nowhere else.

Reply:
That's a good question: 0 should be a valid result, whereas a failure to retrieve any results should be a separate issue. I'll think on this today.

Reply:
I made sure that the MongoDB query could actually return 0, but it was still failing... I figured out that because 0 as a number is falsy, while `app.js` was passing a zero, the subsequent checks on `accountPage.jsx` were reporting the same as if they'd failed.
But "0" as a string is truthy, so converting the passed data from a number to string has allowed 0 to be a valid result.

Now, I just need to replicate this for Google Drive brews...

Reply:
Tested with Google brews, they're now working correctly. I've just pushed a commit that works with either 0 Mongo or Google brews, and still differentiates out the error to the login instructions when listing the Google Drive folder is not permitted.